src/player/Player.php

```diff
@@ -368,8 +368,11 @@ function() : void{
 			}
 		));
 
-		$this->firstPlayed = $nbt->getLong(self::TAG_FIRST_PLAYED, $now = (int) (microtime(true) * 1000));
-		$this->lastPlayed = $nbt->getLong(self::TAG_LAST_PLAYED, $now);
+		$createDateTimeImmutable = static function(string $tag) use ($nbt) : DateTimeImmutable{
+			return new DateTimeImmutable('@' . $nbt->getLong($tag, (int) (new DateTimeImmutable())->format('Uv')) / 1000);
```

Comment:
This weird default seems unnecessary. The `microtime` version was more readable.